src/workflows-app/components/WorkflowsAppNavPanel.ts

```diff
@@ -138,13 +140,22 @@ export const WorkflowsAppNavPanel = ({
           Collapse,
           {
             style: { borderBottom: `1px solid ${colors.dark(0.2)}` },
-            title: span({ style: { color: !pageReady ? colors.disabled() : colors.accent(), fontSize: 15 } }, [
-              'Find & add workflows',
-            ]),
-            initialOpenState: pageReady,
+            title: span(
+              {
+                style: {
+                  color: !pageReady || !workspace.canCompute ? colors.disabled() : colors.accent(),
+                  fontSize: 15,
+                },
+              },
+              ['Find & add workflows']
+            ),
+            tooltip: !workspace.canCompute
+              ? 'You must be a workspace writer/owner to add workflows to this workspace. To import (and run) workflows, you can clone this workspace.'
+              : undefined,
+            initialOpenState: pageReady && workspace.canCompute,
             titleFirst: true,
             summaryStyle: { padding: '1rem 1rem 1rem 1.5rem' },
-            disabled: !pageReady,
+            disabled: !pageReady || !workspace.canCompute,
```

Comment:
For my own clarification, is `workspace.writer` different than `workspace.canCompute`? If it is, maybe we want to check for `workspace.writer` permission rather than compute as I think that would be less restrictive and writers should be able to add workflows to the workspace? Or is is that I am imagining `workspace.writer` permission? 🤔

Reply:
`canCompute` is true for writers, owners, and BP owners, so I've found it easier to check against that than comparing the roles against all 3, or declaring an additional variable to check `canWrite(workspace.accessLevel)`. I also found [this](https://broadinstitute.slack.com/archives/C03GMG4DUSE/p1660846833667199?thread_ts=1660762328.743699&cid=C03GMG4DUSE) which I took to mean that checking `canCompute` is more accurate

Reply:
So based on the Slack message, its possible writers don't have compute permissions. So then they won't be able to import workflows in the workspace. Is that the expected behavior? 🤔 I had assumed that writers without compute permission should be able to import workflows but not be able to run it. Although its possible that in Azure its not possible to have writer permission without compute permission?

Reply:
I think this is an open question within the org because the default assumption ([here](https://github.com/broadinstitute/firecloud-orchestration/blob/0886960a2c2b211c77135b0e2fae53af6e6e24e0/src/main/resources/swagger/api-docs.yaml#L8737)) is that writers can compute, and owners can compute and share, but in GCP there are some cases where writers can't compute, or readers can share - https://broadinstitute.slack.com/archives/C02LMB1PQ6R/p1696958545666059?thread_ts=1696951987.806479&cid=C02LMB1PQ6R.

Reply:
Ah I see. Thank you for the clarification on this. I guess with that assumption it makes sense to simply check for `canCompute` 👍